x-pack/plugins/security/server/authentication/index.ts

```diff
@@ -75,8 +86,15 @@ export async function setupAuthentication({
       encryptionKey: config.encryptionKey,
       isSecure: config.secureCookies,
       name: config.cookieName,
-      validate: (sessionValue: ProviderSession) =>
-        !(sessionValue.expires && sessionValue.expires < Date.now()),
+      validate: (session: ProviderSession) => {
```

Comment:
Should we change the type definition of validate from `session: ProviderSession` to `session: ProviderSession | ProviderSession[]`?

Reply:
Sure. The Hapi Cookie plugin doesn't document that it can actually pass back a single object or an array, which I assume is why it was written this way in the first place.